src/Feature/Checkout/tests/Controllers/CartsControllerTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Feature.Checkout.Tests.Controllers
+{
+    using System;
+
+    using Checkout.Controllers;
+
+    using Foundation.Commerce.Models;
+    using Foundation.Commerce.Services.Cart;
+
+    using Models.Requests;
+
+    using NSubstitute;
+
+    using Ploeh.AutoFixture;
+    using Wooli.Foundation.Commerce.Models.Entities.Cart;
+    using Xunit;
+
+    public class CartsControllerTests
+    {
+        private readonly ICartService cartService;
+
+        private readonly CartsController controller;
+
+        private readonly IFixture fixture;
+
+        public CartsControllerTests()
+        {
+            this.cartService = Substitute.For<ICartService>();
+            this.controller = Substitute.For<CartsController>(this.cartService);
+            this.fixture = new Fixture();
+        }
+
+        [Fact]
+        public void GetCart_ShouldCallExecuteMethod()
+        {
+            // act
+            this.controller.GetCart();
+
+            // assert
+            this.controller.Received(1).Execute(this.cartService.GetCart);
+        }
+
+        // TODO: Tests are valid. Should be uncommented after fixing issue with Quantity validation
```

Comment:
Ok, thanks. Please do not forget add this note to Front-end refactoring task about quantity.

Reply:
Already did